app/uk/gov/hmrc/cataloguefrontend/CatalogueController.scala

```diff
@@ -602,3 +629,17 @@ object DefaultBranchesFilter {
     )(DefaultBranchesFilter.apply)(DefaultBranchesFilter.unapply)
   )
 }
+
+case class PlatformInitiativesFilter(
```

Comment:
OBSERVATION: I think this fits better in the model package, rather than in the Controller file